src/clj/puppetlabs/trapperkeeper/services/protocols/filesystem_watch_service.clj

```diff
@@ -4,8 +4,11 @@
 
 (def Event
   "Schema for an event on a file watched by this service."
-  {:type (schema/enum :create :modify :delete :unknown)
-   :path File})
+  (schema/if #(= (:type %) :unknown)
+    {:type (schema/eq :unknown)
+     :path (schema/eq nil)}
```

Comment:
I don't think I've seen this usage before. I think it's more common to do `(schema/pred nil?)`, but this is equivalent in function, so I don't think it matters.